logstash/datadog_checks/logstash/logstash.py

```diff
@@ -140,11 +145,11 @@ def _get_data(self, url, config, send_sc=True):
         auth = None
 
         # Load SSL configuration, if available.
-        # ssl_verify can be a bool or a string (http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification)
-        if isinstance(config.ssl_verify, bool) or isinstance(config.ssl_verify, str):
+        # ssl_verify can be a bool or a string
+        # (http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification)
+        if isinstance(config.ssl_verify, (bool, str)):
             verify = config.ssl_verify
         else:
-            self.log.error("ssl_verify in the configuration file must be a bool or a string.")
```

Comment:
Let's not remove that, or replace by raising a `ConfigurationError`, WDYT ?

Reply:
The issue was that it's displayed by default as ssl_verify is None. Should I change the default to False? It feels bad to not do security by default.